jep/1/README.adoc

```diff
@@ -0,0 +1,754 @@
+
+|===
+
+
+[[abstract]]
+== Abstract
+
+A Jenkins Enhancement Proposal (JEP) is a design document that
+describes a new feature or aspect of Jenkins itself,
+or the Jenkins Project processes or environment.
+This JEP describes the structure of JEP documents
+and details the process by which JEPs are
+created, submitted, reviewed, finalized, and maintained.
+
+[[specification]]
+== Specification
+
+=== What is a JEP?
+
+JEP stands for Jenkins Enhancement Proposal.
+A JEP is a design document that
+describes a new feature or aspect of Jenkins itself, or Jenkins project processes or environment.
+A JEP provides a concise technical specification of the feature,
+describes the motivation for the change,
+and discusses the rationale behind the design.
+
+JEPs are the primary mechanisms for proposing major new
+features, for collecting community input on an issue, and for
+documenting the design decisions that have gone into Jenkins.
+Each JEP has an <<Author>>.
+The author of a JEP may or may not actually write the text of the JEP.
+Regardless, the JEP Author is responsible for building
+consensus for that JEP within the community and
+documenting dissenting opinions.
+
+Because the JEPs are maintained as text files in a versioned
+repository, their revision history is the historical record of the
+feature proposal footnoteref:[repo, https://github.com/jenkinsci/jep].
+
+
+=== JEP Types
+
+There are three kinds of JEP:
+
+. A **Standards Track** JEP describes a new feature or implementation
+ for Jenkins. It may also describe an interoperability or
+ backwards-compatibility standard which will be supported for the feature in
+ current versions of Jenkins, moving forward.
+. An **Informational** JEP describes a Jenkins design issue, or
+ provides general guidelines or information to the Jenkins community,
+ but does not propose a new feature. Informational JEPs do not
+ necessarily represent a Jenkins community consensus or
+ recommendation, so users and implementers are free to ignore
+ Informational JEPs or follow their advice.
+. A **Process** JEP describes a process surrounding Jenkins, or
+ proposes a change to (or an event in) a process. Process JEPs are like
+ Standards Track JEPs but apply to areas other than the Jenkins codebase
+ itself. They may propose an implementation, but not for what would be
+ generally considered the Jenkins codebase; they often require community
+ consensus; unlike Informational JEPs, they are more than recommendations, and
+ users are typically not free to ignore them. Examples include procedures,
+ guidelines, changes to the decision-making process, and changes to the tools
+ or environment used in Jenkins development. Any meta-JEP (JEP focusing on the
+ improvement of the Jenkins Enhancement Proposal process) is also considered a
+ Process JEP.
+
+=== JEP Terminology
+
+==== BDFL
+
+There are several references in this JEP to the "BDFL". This acronym stands for
+"Benevolent Dictator for Life", and refers to
```

Comment:
> I will remind you that despite what some contributors seem to think, in the US "KKK" refers to a hateful group of racists and domestic terrorists.

Well, this is my turn to apologize then. My only excuse is that I am not an American citizen/resident and not a native speaker. 

Regarding this group, I know Ku Klux Klan only by the full name, and actually I have not really thought that "Klan" starts from "K" instead of "C". So for me this abbreviation never meant this group which I definitely do not support. And I definitely do not find this abbreviation funny after the clarification (probably I have missed the previous one).

I will stop using this abbreviation for sure. Unfortunately such mistakes happen too often in a global world, especially in the case case of acronyms.

Reply:
RESOLVED.